docs/source/installing-brave.rst

```diff
@@ -14,6 +14,21 @@ Release Channel Installation
 
 .. highlight:: console
 
+Debian
+--------------------------
+::
+
+    curl -s https://brave-browser-apt-release.s3.brave.com/brave-core.asc | sudo apt-key --keyring /etc/apt/trusted.gpg.d/brave-browser-release.gpg add -
+
+    source /etc/os-release && CODENAME=$(echo $VERSION | rev | cut -c 2- | rev | cut -d"(" -f 2)
+
+    echo "deb [arch=amd64] https://brave-browser-apt-release.s3.brave.com/ $CODENAME main" | sudo tee /etc/apt/sources.list.d/brave-browser.list
```

Comment:
This is so similar it's a shame we can't deduplicate instructions here. I guess we will be able to when we switch this to the `stable` suite.

Reply:
Agreed. We should just start building for `stable` and change all the instructions to that.